pkg/czid/uploadSamples.go

```diff
@@ -55,6 +56,14 @@ type UploadInfo struct {
 	S3Path            string  `json:"s3_path"`
 }
 
+// Must match file type constants present in CZID's InputFile model
+type FileType string
+const (
+	FASTQFileType FileType = "fastq"
+	PrimerBedFileType FileType = "primer_bed"
+	ReferenceAccessionFileType FileType = "ref_seq"
```

Comment:
```suggestion
	ReferenceAccessionFileType FileType = "reference_sequence"
```

Reply:
nit: the Reference file isn't always going to be an accession file, could we rename this to `ReferenceSequenceFileType`?